frontend/src/__tests__/cypress/cypress/pages/modelRegistry.ts

```diff
@@ -30,6 +30,14 @@ class ModelRegistry {
     return this;
   }
 
+  registeredModelsEmpty() {
+    cy.findByTestId('no-registered-models').should('exist');
+  }
```

Comment:
This function performs assertion
```suggestion
  shouldRegisteredModelsEmpty() {
    cy.findByTestId('no-registered-models').should('exist');
    return this;
  }
```